lib/Workflow/Config/XML.pm

```diff
@@ -44,10 +44,20 @@ my %XML_OPTIONS = (
 );
 
 my $XML_REQUIRED = 0;
+my $warned = 0;
```

Comment:
Instead of this, please see [Deprecated and its recommendations](https://metacpan.org/pod/Dist::Zilla::Plugin::Deprecated#Recommendations).